standalone/test/unit_tests/api/unittest_laserscan.cpp

```diff
@@ -27,54 +28,121 @@ namespace psen_scan_v2_standalone_test
 static const util::TenthOfDegree DEFAULT_RESOLUTION{ 1 };
 static const util::TenthOfDegree DEFAULT_START_ANGLE{ 1 };
 static const util::TenthOfDegree DEFAULT_END_ANGLE{ 2 };
+static const int64_t DEFAULT_TIMESTAMP{ 1 };
+static const uint32_t DEFAULT_SCAN_COUNTER{ 1 };
+
+class LaserScanBuilder
+{
+public:
+  LaserScan build()
+  {
+    return LaserScan(resolution_, start_angle_, end_angle_, scan_counter_, timestamp_);
+  }
+
+  LaserScanBuilder& resolution(const util::TenthOfDegree& resolution)
+  {
+    resolution_ = resolution;
+    return *this;
+  }
+
+  LaserScanBuilder& startAngle(const util::TenthOfDegree& start_angle)
+  {
+    start_angle_ = start_angle;
+    return *this;
+  }
+
+  LaserScanBuilder& endAngle(const util::TenthOfDegree& end_angle)
+  {
+    end_angle_ = end_angle;
+    return *this;
+  }
+
+  LaserScanBuilder& scanCounter(const uint32_t scan_counter)
+  {
+    scan_counter_ = scan_counter;
+    return *this;
+  }
+
+  LaserScanBuilder& timestamp(const int64_t timestamp)
+  {
+    timestamp_ = timestamp;
+    return *this;
+  }
+
+private:
+  util::TenthOfDegree resolution_{ DEFAULT_RESOLUTION };
+  util::TenthOfDegree start_angle_{ DEFAULT_START_ANGLE };
+  util::TenthOfDegree end_angle_{ DEFAULT_END_ANGLE };
+  uint32_t scan_counter_{ DEFAULT_SCAN_COUNTER };
+  int64_t timestamp_{ DEFAULT_TIMESTAMP };
+};
 
 TEST(LaserScanTests, testInvalidZeroResolution)
 {
-  EXPECT_THROW(LaserScan laser_scan(util::TenthOfDegree(0), DEFAULT_START_ANGLE, DEFAULT_END_ANGLE),
-               std::invalid_argument);
+  LaserScanBuilder builder;
+  builder.resolution(util::TenthOfDegree(0));
+  EXPECT_THROW(builder.build(), std::invalid_argument);
 }
 
 TEST(LaserScanTests, testOutOfRangeResolution)
 {
-  EXPECT_THROW(LaserScan laser_scan(util::TenthOfDegree(10000), DEFAULT_START_ANGLE, DEFAULT_END_ANGLE),
-               std::invalid_argument);
+  LaserScanBuilder builder;
+  builder.resolution(util::TenthOfDegree(10000));
+  EXPECT_THROW(builder.build(), std::invalid_argument);
```

Comment:
would call it laserScanBuilder or lsBuilder or something that gives the EXPECT something more meaningful than builder.build()